llvm/test/Transforms/LoopRotate/rotate-oz-if-branch-fold.ll

```diff
@@ -0,0 +1,39 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -S -Oz -debug -debug-only=loop-rotate 2>&1 | FileCheck %s -check-prefix=OZ
```

Comment:
If you want to test `-Oz`, this must go into the PhaseOrdering directory -- and you also need dedicated tests for only LoopRotate. Also, do not test debug output.